### DISCUSSION TITLE
Terminal windows in the background

### DISCUSSION THREAD

**Discussion by Aggelos11** [caido/caido#1622] [Q&A]
Is the terminal windows that runs in the background necessary to exist ? I believe it would be better if it did not 
![IMG_1098](https://github.com/user-attachments/assets/c438775e-6985-4d6b-9931-228790bb6787)

**Comment by Sytten** [comment]
There are multiple ways to not have that window. Either you can use the desktop application, you can spawn the process in detached mode (with the `&` at the end like any other process), you can use a systemd service, etc.
